chia/consensus/blockchain.py

```diff
@@ -286,17 +290,37 @@ async def receive_block(
         if fork_height is not None:
             # new coin records added
             assert coin_record_change is not None
-            return ReceiveBlockResult.NEW_PEAK, None, fork_height, coin_record_change
+            return ReceiveBlockResult.NEW_PEAK, None, fork_height, (coin_record_change, hint_changes)
         else:
-            return ReceiveBlockResult.ADDED_AS_ORPHAN, None, None, []
+            return ReceiveBlockResult.ADDED_AS_ORPHAN, None, None, ([], {})
+
+    def get_hint_list(self, npc_result: NPCResult) -> List[Tuple[bytes, bytes]]:
+        h_list = []
+        for npc in npc_result.npc_list:
+            for opcode, conditions in npc.conditions:
+                if opcode == ConditionOpcode.CREATE_COIN:
+                    for condition in conditions:
+                        if len(condition.vars) > 2 and condition.vars[2] != b"":
+                            puzzle_hash, amount_bin = condition.vars[0], condition.vars[1]
```

Comment:
@Quexington was saying that the 3rd argument should be a list, so we can add more things there later. This is also how I am setting the memo for retirement of CAT tokens: 
https://github.com/Chia-Network/chia-blockchain/pull/8600

Reply:
I think this needs to be changed in clvm_rs first, was there change there?

Reply:
Just saw Arvids pr is open, I'll have to wait for that to get in & clvm_rs pypy release is available.